docs/alerts-and-notifications/notifications/centralized-cloud-notifications/manage-notification-methods.md

```diff
@@ -31,19 +31,19 @@ To manage Space notification settings, you will need the following:
 2. Click on the **Alerts & Notifications** tab on the left-hand side.
 3. Click on the **Notification Methods** tab.
 4. You will be presented with a table of the configured notification methods for the Space. You will be able to:
-    1. **Add a new** notification method configuration.
+    - **Add a new** notification method configuration.
         - Choose the service from the list of available ones. The available options will depend on your subscription plan.
         - You can optionally provide a name for the configuration so you can refer to it.
         - You can define the filtering criteria, regarding which Rooms the method will apply, and what notifications you want to receive (All Alerts and unreachable, All Alerts, Critical only).
```

Comment:
This one should also have the same options as the notification selection / filtering with the personal (email/ mobile app) notification settings.